presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -362,7 +372,7 @@ private void printQueryInfo(QueryResults results)
 
     private void printStageTree(StageStats stage, String indent, AtomicInteger stageNumberCounter)
     {
-        Duration elapsedTime = nanosSince(start);
+        long now = System.nanoTime();
```

Comment:
Just wondering if it will be better to pass the `now` nanoTime as a argument to this function so that every sub stage will have the same elapsed Time. Thanks.